.github/stale.yml

```diff
@@ -0,0 +1,27 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# General configuration
+# Label to use when marking as stale
+staleLabel: stale
+
+
+# Issue specific configuration
+issues:
+  # TODO: Raise the limitPerRun once we are satisfied with the bot's performance
+  limitPerRun: 1
+  daysUntilStale: 14
+  daysUntilClose: 14
+  markComment: >
+    This issue has been automatically marked as stale because it has not had activity in the
+    last 14 days. It will be closed in the next 14 days if no further activity occurs.
+    Thank you for your contributions.
+  closeComment: >
+    This issue has been automatically closed because it has not had activity in the
+    last 28 days. If this issue is still valid, please ping a maintainer and ask them to relabel it.
```

Comment:
Does re-opening a closed issue reset the timer on the stalebot? What scenario do you have in mind for the proposal "please ping a maintainer and ask them to relabel it."

Reply:
I think it's worthwhile to give a user recourse if they think the issue has been closed in error. Especially if they provided information and are waiting for a response from us.

I adapted some of the wording here from https://github.com/envoyproxy/envoy/blob/master/.github/stale.yml

Via Google search, some other configs have similar wordings:
https://github.com/rails/rails/blob/master/.github/stale.yml
```
  This issue has been automatically marked as stale because it has not been commented on for at least three months.

  The resources of the Rails team are limited, and so we are asking for your help.

  If you can still reproduce this error on the `6-0-stable` branch or on `master`,
  please reply with all of the information you have about it in order to keep the issue open.

  Thank you for all your contributions.
```
https://www.codefactor.io/repository/github/facebook/create-react-app/source/master/.github/stale.yml
```
This issue has been automatically closed because it has not had any recent
activity. If you have a question or comment, please open a new issue.
```
We can also apparently use `closeComment: false` and it won't say anything, but I'd be annoyed by that as a user.